scripts/release.minor.rst

```diff
@@ -3,8 +3,8 @@ pytest-{version}
 
 The pytest team is proud to announce the {version} release!
 
-This release contains new features, improvements, bug fixes, and breaking changes, so users
-are encouraged to take a look at the CHANGELOG carefully:
+This release contains new features, improvements, and bug fixes,
+A full list of changes is available in the changelog:
```

Comment:
```suggestion
the full list of changes is available in the changelog:
```

The capitalize `A` here is wrong because the previous line ends in a comma, so this should be either `a` (or `the`?),  or a new phrase.

Reply:
Good catch, thanks!